.github/workflows/build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM hyperledger/indy-core-baseci:0.0.3-master
+LABEL maintainer="Hyperledger <hyperledger-indy@lists.hyperledger.org>"
+
+RUN apt-get update -y && apt-get install -y \
+    python3-nacl \
+    libindy-crypto=0.4.5 \
+    libindy=1.13.0~1420 \
```

Comment:
TODO: bump to 1.15.0

Reply:
@Toktar, @m00sey did try this, but it caused errors in Jenkins build; https://github.com/hyperledger/indy-node/pull/1645#discussion_r571456440

Reply:
Then I suggest bumping libindy and libindy's wrapper to the last master because it's a bad idea to have different versions. But we can do it in the next PR.

Reply:
@WadeBarnes How can it affect Jenkins? Maybe this version is just missing in the repository?

Reply:
Likely, I was going off the related Jenkins build error from this change; https://github.com/hyperledger/indy-node/pull/1645#discussion_r571456440